src/ert/shared/models/base_run_model.py

```diff
@@ -535,7 +535,10 @@ def check_if_runpath_exists(self) -> bool:
         idx = [i for i in range(len(active_mask)) if active_mask[i]]
         for i in range(start_iteration, number_of_iterations):
             for j in idx:
-                s = run_path % (j, i)
+                if run_path.count("%d") == 2:
```

Comment:
It is very hard to understand why this is done the way it is. Could you please add some documentation about what is going on.

Reply:
Added a comment to the function to clarify the run_path format